### PR TITLE
split save into save and publish

### PR DESCRIPTION
**Pull request by thejohnhoffer**
- This should make no difference to the behavior of the cloud version
- This splits the /save and /render local API to save and publish buttons

### CODE REVIEW

File: src/containers/repo.jsx [hunk 1]
```diff
@@ -1969,15 +1998,28 @@ class Repo extends Component {
...
     );
     if (this.props.env === 'local') {
       // Hide buttons which are not implemented in local environment yet
       // TODO - Implement rendering in backend and show previewButton 
       previewButton = null;
       shareButton = null;
       publishButton = null;
-      if (this.state.saved) {
+      if (group != undefined) {
+        publishButton = (
```
Comment (Juha-Ruokonen) [inline src/containers/repo.jsx]: I'd rather not duplicate the publishButton code, but have just one button and then branch out the logic in this.publish() method (based on whether we're running the local or the cloud version of the UI)

Comment (thejohnhoffer) [reply src/containers/repo.jsx]: How should we handle the publish button when the group is undefined?

I'm not sure how Minerva Cloud handles this, but for the local version it only makes sense to publish if there is at least one channel group, so I would favor only showing the publish button if there is an active channel group.

Comment (thejohnhoffer) [reply src/containers/repo.jsx]: I've just set the publish button to only appear if there is an active group.

Comment (Juha-Ruokonen) [reply src/containers/repo.jsx]: I agree, that's a sensible approach 👍

**Comment by Juha-Ruokonen** [comment]
Looking good now!

**Pull request merged**